docker/Dockerfile.ldk

```
@@ -0,0 +1,49 @@
+FROM ubuntu:22.04
```

```suggestion
FROM ubuntu:25.04
```

Do you think that we can upgrade to the last docker image?

---

I tried it but Ubuntu 25.04 ships with Python 3.13, which isn’t yet supported by Poetry’s installer. Poetry currently supports Python versions up to 3.12.

If we want to do it, we will have to build python 3.12 to run with poetry.